src/main/java/org/fungover/haze/HazeList.java

```diff
@@ -13,6 +14,31 @@ public class HazeList {
 
     public HazeList(HazeDatabase hazeDatabase) {
         this.hazeDatabase = hazeDatabase;
+        this.elements = new ArrayList<>();
```

Comment:
Still saving the list in a local arraylist instead of storing it into the hazeDatabase?

Reply:
i changed it so it should update on my next commit and push